app/assets/javascripts/miq_explorer.js

```diff
@@ -14,22 +14,77 @@ ManageIQ.explorer.buildCalendar = function(options) {
   ManageIQ.calendar.calSkipDays = _.isObject(options.skipDays) ? options.skipDays : undefined;
 
   miqBuildCalendar();
-}
+};
 
 ManageIQ.explorer.lock_tree = function(tree, lock) {
   $('#' + tree + 'box').dynatree(lock ? 'disable' : 'enable');
   miqDimDiv('#' + tree + '_div', lock);
-}
+};
 
-ManageIQ.explorer.clear_search_toggle = function(show) {
+ManageIQ.explorer.clearSearchToggle = function(show) {
   if ( show ) {
     $('#clear_search').show();
   } else {
     $('#clear_search').hide();
   }
-}
+};
 
 ManageIQ.explorer.process = function(data) {
+  switch (data.explorer) {
+    case 'flash':
+      ManageIQ.explorer.processFlash(data);
+      break;
+    case 'replace_right_cell':
+      ManageIQ.explorer.processReplaceRightCell(data);
+      break;
+    case 'replace_main_div':
+      ManageIQ.explorer.processReplaceMainDiv(data);
+      break;
+    case 'buttons':
+      ManageIQ.explorer.processButtons(data);
+      break;
+  }
+};
+
+ManageIQ.explorer.processButtons = function(data) {
+  ManageIQ.explorer.miqButtons(data);
+};
+
+ManageIQ.explorer.processReplaceMainDiv = function(data) {
+  ManageIQ.explorer.updatePartials(data);
+}
+
+ManageIQ.explorer.processFlash = function(data) {
+  ManageIQ.explorer.replacePartials(data);
+}
+
+ManageIQ.explorer.replacePartials = function (data) {
```

Comment:
There's extra whitespace before `(data)`, which needs to go away (there also are other occurences of this problem in this PR)